ethereum/core/src/main/java/org/hyperledger/besu/ethereum/chain/DefaultBlockchain.java

```diff
@@ -50,6 +51,7 @@
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.collect.Lists;
 import com.google.common.collect.Streams;
+import org.jetbrains.annotations.NotNull;
```

Comment:
accidental import?

Reply:
it isn't clear whether this was here for static analysis during development or if it was intended as a runtime validation.  Regardless, the ci/cd pipeline doesn't do anything with intellij static analysis annotations, so I will remove this and we can circle back with @fab-10 to see if he intended to use javax.annotation.* or javax.validation.constraints.*